src/sonic-yang-models/tests/yang_model_tests/tests_config/port.json

```diff
@@ -78,10 +82,28 @@
                         "alias": "eth8",
                         "lanes": "65",
                         "speed": 25000,
+                        "tpid": "0x8100",
                         "autoneg": 0
                     }
                 ]
             }
         }
-    }
+    },
+
+    "PORT_INVALID_TPID_TEST": {
```

Comment:
We do not need extra test, once type is moved in stypes.

Reply:
Not sure I understood your comment.  moving the tpid type to a common typedef allows sharing of the same definition  but testing is still required isn't it?  LThis was added so that we can see that the Yang model correctly catches any attempt of using a wrong TPID value that is not specified in the model.  Please clarify ...
Thanks!